crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -471,8 +500,36 @@ fn create_get_pending_operations_message() -> Result<Message, ConversionError> {
     Ok(Message::new(&topic, payload))
 }
 
-fn create_supported_log_types_message() -> Result<Message, ConversionError> {
-    let payload = SmartRestSetSupportedLogType::default().to_smartrest()?;
+fn get_supported_log_types(log_dir: &Path) -> Result<Vec<String>, ConversionError> {
```

Comment:
When there are no log files, how the types will be derived?. This log type derivation is dynamic? or just updated when the mapper starts?. If it's the latter case then it's problematic, right?

Reply:
> This log type derivation is dynamic?

only when the mapper starts but #1077 will make it dynamic.